homeassistant/components/iaqualink/__init__.py

```diff
@@ -111,14 +114,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         _LOGGER.warning("Exception raised while attempting to login: %s", aio_exception)
         raise ConfigEntryNotReady from aio_exception
 
-    systems = await aqualink.get_systems()
+    try:
+        systems = await aqualink.get_systems()
+    except AqualinkServiceException as svc_exception:
+        _LOGGER.warning(
+            "Exception raised while attempting to retrieve systems list: %s",
+            svc_exception,
+        )
```

Comment:
```suggestion
```

Drop this so it doesn't log over and over during retry, and add it to `ConfigEntryNotReady` so it shows in the UI